src/es_archiver/cli.js

```diff
@@ -30,7 +30,9 @@ cmd
     console.log(readFileSync(resolve(__dirname, './cli_help.txt'), 'utf8'));
   });
 
-cmd.command('save <name> <indices...>')
+cmd
+  .option('--raw', `don't gzip the archive`)
+  .command('save <name> <indices...>')
   .description('archive the <indices ...> into the --dir with <name>')
   .action((name, indices) => execute('save', name, indices));
```

Comment:
I think I'd prefer that we pass `{ raw: !!cmd.raw }` as the final argument here, rather than always passing `cmd` as the last argument in `execute()`.

Reply:
Hm. I see where you're coming from. It's dirty to pass the entire cmd into functions that are expecting something pretty different. But the alternative is putting save-specific logic in a place that currently serves more functions than just save. Hm. Both approaches seem dirty to me :/

Let's see if @tylersmalley has an opinion.